src/components/axes/timeAxis.ts

```diff
@@ -141,6 +141,8 @@ export module Axis {
     private _tierMarkContainers: D3.Selection[];
     private _tierBaselines: D3.Selection[];
     private _tierHeights: number[];
+    private _currentTimeAxisConfigurations: TimeAxisConfiguration[];
```

Comment:
Why was the name of this variable changed? "current" is probably not the right description here...